Ft/bookStore-setup

zbahati opened the pull request:
- Add linters
- Add components
- Add Routes Folder that containes Route pages
- Render all to the App.js file to display to the screen
- Add one style to remove lebel of button to fix linters error

[#Finishes ..]

Commit:
Ft/bookStore-setup

- Add linters
- Add components
- Add Routes Folder that containes Route pages
- Render all to the App.js file to display to the screen
- Add one style to remove lebel of button to fix linters error

[#Finishes ..]

Commit:
Add Readme and License

rloterh reviewed (commented):
Hi Bahati, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

Kindly ensure that your ReadMe is professionally documented; Remove the Live demo link from the ReadMe if the app is not yet deployed.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

zbahati committed:
Update README.md

- Remove Live Demo to the ReadMe

tufoinnkuo10 commented on README.md:
```diff
@@ -1 +1,215 @@
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements">Microverse</a>
+
```
- [optional] you can remove one acknowledgement header and write everything under one. this will make your Readme professional.

tufoinnkuo10 commented on src/components/FormInput.js:
```diff
@@ -0,0 +1,10 @@
+const NewBook = () => (
+  <div className="newbook">
+    <form>
+      <input type="text" placeholder=" New Book" />
```
- [ ] Great job so far. Remember, you will be working on creating a complete book with its author. Kindly ensure it Renders a form with inputs for the book title and the author of the book

tufoinnkuo10 reviewed (changes requested):
Hi @zbahati 

There are some issues that you still need to work on to go to the next project but you are almost there!

To Highlight
✅ You used the correct Gitflow
✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally


Required Changes ♻️
Check the comments under the review
Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

Commit:
Add changes Requested

- Remove One Aknowledgement title and combine the two in one title
- Add other input for Author

[#Finishes ..]

Commit:
Merge branch 'bookStore-setup' of https://github.com/zbahati/BookStore-React into bookStore-setup

OyePriscilla reviewed (approved):
Hi @zbahati 👏 👏, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
![200](https://user-images.githubusercontent.com/103386183/196424328-985f6e98-f7c6-4cf9-bafe-158436f70560.gif)

## Highlights

Implemented a form with all the necessary input elements ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

OyePriscilla reviewed (approved):
Hi @zbahati 👏 👏, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
![200](https://user-images.githubusercontent.com/103386183/196424328-985f6e98-f7c6-4cf9-bafe-158436f70560.gif)

## Highlights

Implemented a form with all the necessary input elements ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
